layouts/partials/header_home.html

```diff
@@ -1,7 +1,7 @@
 {{ $current := . }}
 {{ $list := .Site.Menus.main | append .Site.Menus.header }}
-  <nav class="fixed lg:absolute top-0 left-0 w-full z-50">
-    <div class="container mx-auto">
+  <nav class="fixed lg:absolute top-0 left-0 w-full z-50 bg-card-light lg:bg-transparent">
+    <div class="responsive-container">
```

Comment:
these two lines are the same changes from #176. I needed to add it here for me to test